zerver/migrations/0307_realm_online_mention_policy.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.17 on 2020-12-22 21:00
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("zerver", "0306_custom_profile_field_date_format"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="realm",
+            name="online_mention_policy",
+            field=models.PositiveSmallIntegerField(default=4),
+        ),
+    ]
```

Comment:
The migrations file changes in this PR aren't correct; see https://zulip.readthedocs.io/en/latest/subsystems/schema-migrations.html for how to renumber your migrations properly when rebasing (which I assume is what went wrong here).